Ginger/GingerCore/ALM/RQMCore.cs

```diff
@@ -304,6 +308,20 @@ private string GetIsTestSuiteValueFromDict(Dictionary<string, object> dic)
                 return "";
             }
         }
+
+        private string GetIsSkippedUpdateValueFromDict(Dictionary<string, object> dic)
+        {
+            if (dic.ContainsKey("IsSkippedUpdate"))
+            {
+                return (string)dic["IsSkippedUpdate"];
+            }
+            else
+            {
+                return "";
```

Comment:
Return should be false instead of empty string as it will fail in parsing

Reply:
Updated